torch/csrc/distributed/c10d/ProcessGroupUCC.cpp

```diff
@@ -1619,7 +1619,7 @@ void ProcessGroupUCC::initComm(c10::Device dev) {
   // Create UCC execution engine.
   if (!cuda_ee && dev.is_cuda()) {
     stream = std::make_unique<at::cuda::CUDAStream>(
-        at::cuda::getStreamFromPool(true, dev.index()));
+        at::cuda::getStreamFromPool(-1, dev.index()));
```

Comment:
It might be better to add a new API that takes the integers, and have the old one around as a BC. It's also sort of strange that 0 is low priority and -1 is high priority, curious what the motivation here is.

Reply:
Cool, I'll do that (that was my question), what do you think old `getStreamFromPool` with `isHighPriority=true` calls should translate to in the new world? Highest priority or one notch above the default?
`-1` and lower numbers is how cuda APIs expect priority values, and also how current public pytorch APIs work (`priority_range` returns 0, -1 currently with `-1` denoting high priorty, `priority` argument to stream should be the value returned by `priority_range`, so `-1` again https://pytorch.org/docs/stable/generated/torch.cuda.Stream.html#torch.cuda.Stream

Reply:
I would arrange for it to send the same priority to the CUDA APIs that it did before, in case someone was manually calling into the APIs.